### DISCUSSION TITLE
Relação entre Competência e Perfil.

### DISCUSSION THREAD

**Discussion by NathanJFA** [universi-me/universi-api#29] [Ideas]
Como foi acordado em aula, ficamos de maturar a ideia desse mapeamento.
Se é muitos para muitos, ou se é uma competência está vinculada apenas a um perfil.

Tenho essa incognita pois se permitimos que cada um insira um nome diferente para uma competência, na hora de fazer a pesquisa não vamos conseguir puxar todos os dados, exemplo: (JavaScript, Java Script, javascript, Javascript). 

Talvez o nome precise ser um aributo que uma vez gerado, ficasse como opção para as próximas seleções.

Deixem ai suas ideias.

**Comment by adriannfernandes** [comment]
Na minha opnião a proposta é sim conveniente, mas não acho que seja uma preocupação para o momento.
